low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -208,13 +201,7 @@ class SSPLDevDeploy:
         """Install dependencies."""
         pkg_name = "cortx-py-utils"
         print("INFO: INSTALLING cortx-py-utils...")
-        print("Downloading pyutils from %s" % self.build_url)
-        pkg = [re.findall("<a .*>(.*?)</a>", str(line))[0]
-               for line in urllib.request.urlopen(self.cortx_utils_build_url).readlines()
-               if pkg_name in str(line)]
-        if not pkg:
-            print(f"ERROR: Unable to download {pkg_name}.")
-        cmd = "yum install -y %s%s" % (self.cortx_utils_build_url, pkg[0])
+        cmd = "yum install -y --nogpgcheck cortx-py-utils"
```

Comment:
You added `--nogppcheck` for installing cortx-py-utils? Why?

Reply:
We are no longer pulling cortx-py-utils from remote. We build it locally and we don't add gpg sign. so all local RPMs installation will have --nogpgcheck. Same thing is mentioned in cortx build guide (https://github.com/Seagate/cortx/blob/main/doc/community-build/ProvisionReleaseBuild.md).

Reply:
Okay.